{
  "uid" : "ec8a50870930f523",
  "name" : "NHS Number: Searching with invalid NHS number shows an error [123456789-Enter 10 digits]",
  "fullName" : "/app/features/find_a_patient.feature:NHS Number: Searching with invalid NHS number shows an error",
  "historyId" : "f44352c612938e33e618f3f1b20aabf5",
  "time" : {
    "start" : 1731572127672,
    "stop" : 1731572302718,
    "duration" : 175046
  },
  "status" : "failed",
  "statusMessage" : "AssertionError: assert False is True\n +  where False = check_nhs_number_error_message_text_exists('Enter 10 digits')",
  "statusTrace" : "fixturefunc = <function step_error_message_appears_for_nhs_number at 0x7f35c9245120>\nrequest = <FixtureRequest for <Function test_nhs_number_searching_with_invalid_nhs_number_shows_an_error[123456789-Enter 10 digits]>>\nkwargs = {'errorMessage': 'Enter 10 digits'}\n\n    def call_fixture_func(\n        fixturefunc: _FixtureFunc[FixtureValue], request: FixtureRequest, kwargs\n    ) -> FixtureValue:\n        if is_generator(fixturefunc):\n            fixturefunc = cast(\n                Callable[..., Generator[FixtureValue, None, None]], fixturefunc\n            )\n            generator = fixturefunc(**kwargs)\n            try:\n                fixture_result = next(generator)\n            except StopIteration:\n                raise ValueError(f\"{request.fixturename} did not yield a value\") from None\n            finalizer = functools.partial(_teardown_yield_fixture, fixturefunc, generator)\n            request.addfinalizer(finalizer)\n        else:\n            fixturefunc = cast(Callable[..., FixtureValue], fixturefunc)\n>           fixture_result = fixturefunc(**kwargs)\n\n.tox/py311/lib/python3.11/site-packages/_pytest/fixtures.py:898: \n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ \n\nerrorMessage = 'Enter 10 digits'\n\n    @then(parse('I can see an nhs number error message {errorMessage}'))\n    def step_error_message_appears_for_nhs_number(errorMessage):\n        attach_screenshot(\"error_message_appears_for_nhs_number\")\n>       assert check_nhs_number_error_message_text_exists(errorMessage) is True\nE       AssertionError: assert False is True\nE        +  where False = check_nhs_number_error_message_text_exists('Enter 10 digits')\n\ntests/test_find_a_patient_steps.py:130: AssertionError",
  "flaky" : false,
  "newFailed" : false,
  "newBroken" : false,
  "newPassed" : false,
  "retriesCount" : 0,
  "retriesStatusChange" : false,
  "beforeStages" : [ ],
  "testStage" : {
    "status" : "failed",
    "statusMessage" : "AssertionError: assert False is True\n +  where False = check_nhs_number_error_message_text_exists('Enter 10 digits')",
    "statusTrace" : "fixturefunc = <function step_error_message_appears_for_nhs_number at 0x7f35c9245120>\nrequest = <FixtureRequest for <Function test_nhs_number_searching_with_invalid_nhs_number_shows_an_error[123456789-Enter 10 digits]>>\nkwargs = {'errorMessage': 'Enter 10 digits'}\n\n    def call_fixture_func(\n        fixturefunc: _FixtureFunc[FixtureValue], request: FixtureRequest, kwargs\n    ) -> FixtureValue:\n        if is_generator(fixturefunc):\n            fixturefunc = cast(\n                Callable[..., Generator[FixtureValue, None, None]], fixturefunc\n            )\n            generator = fixturefunc(**kwargs)\n            try:\n                fixture_result = next(generator)\n            except StopIteration:\n                raise ValueError(f\"{request.fixturename} did not yield a value\") from None\n            finalizer = functools.partial(_teardown_yield_fixture, fixturefunc, generator)\n            request.addfinalizer(finalizer)\n        else:\n            fixturefunc = cast(Callable[..., FixtureValue], fixturefunc)\n>           fixture_result = fixturefunc(**kwargs)\n\n.tox/py311/lib/python3.11/site-packages/_pytest/fixtures.py:898: \n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ \n\nerrorMessage = 'Enter 10 digits'\n\n    @then(parse('I can see an nhs number error message {errorMessage}'))\n    def step_error_message_appears_for_nhs_number(errorMessage):\n        attach_screenshot(\"error_message_appears_for_nhs_number\")\n>       assert check_nhs_number_error_message_text_exists(errorMessage) is True\nE       AssertionError: assert False is True\nE        +  where False = check_nhs_number_error_message_text_exists('Enter 10 digits')\n\ntests/test_find_a_patient_steps.py:130: AssertionError",
    "steps" : [ {
      "name" : "Given I am on the find a patient by nhs number page",
      "time" : {
        "start" : 1731572127672,
        "stop" : 1731572249956,
        "duration" : 122284
      },
      "status" : "passed",
      "steps" : [ ],
      "attachments" : [ ],
      "parameters" : [ ],
      "stepsCount" : 0,
      "attachmentsCount" : 0,
      "shouldDisplayMessage" : false,
      "hasContent" : false
    }, {
      "name" : "And I enter 123456789 as the nhs number",
      "time" : {
        "start" : 1731572249957,
        "stop" : 1731572270237,
        "duration" : 20280
      },
      "status" : "passed",
      "steps" : [ ],
      "attachments" : [ ],
      "parameters" : [ ],
      "stepsCount" : 0,
      "attachmentsCount" : 0,
      "shouldDisplayMessage" : false,
      "hasContent" : false
    }, {
      "name" : "When I click the search button",
      "time" : {
        "start" : 1731572270237,
        "stop" : 1731572292523,
        "duration" : 22286
      },
      "status" : "passed",
      "steps" : [ ],
      "attachments" : [ ],
      "parameters" : [ ],
      "stepsCount" : 0,
      "attachmentsCount" : 0,
      "shouldDisplayMessage" : false,
      "hasContent" : false
    }, {
      "name" : "Then I can see an nhs number error message Enter 10 digits",
      "time" : {
        "start" : 1731572292524,
        "stop" : 1731572302718,
        "duration" : 10194
      },
      "status" : "failed",
      "statusMessage" : "assert False is True\n +  where False = check_nhs_number_error_message_text_exists('Enter 10 digits')",
      "statusTrace" : "AssertionError: assert False is True\n +  where False = check_nhs_number_error_message_text_exists('Enter 10 digits')\n",
      "steps" : [ ],
      "attachments" : [ {
        "uid" : "a8bba779bbdf993c",
        "name" : "dev_edge_130.0.2849.80_error_message_appears_for_nhs_number.png",
        "source" : "a8bba779bbdf993c.png",
        "type" : "image/png",
        "size" : 4331
      } ],
      "parameters" : [ ],
      "stepsCount" : 0,
      "attachmentsCount" : 1,
      "shouldDisplayMessage" : true,
      "hasContent" : true
    } ],
    "attachments" : [ {
      "uid" : "c68366b31c42f053",
      "name" : "log",
      "source" : "c68366b31c42f053.txt",
      "type" : "text/plain",
      "size" : 1806
    } ],
    "parameters" : [ ],
    "stepsCount" : 4,
    "attachmentsCount" : 2,
    "shouldDisplayMessage" : true,
    "hasContent" : true
  },
  "afterStages" : [ ],
  "labels" : [ {
    "name" : "host",
    "value" : "65c7fd734071"
  }, {
    "name" : "thread",
    "value" : "1102-MainThread"
  }, {
    "name" : "framework",
    "value" : "pytest-bdd"
  }, {
    "name" : "language",
    "value" : "cpython3"
  }, {
    "name" : "feature",
    "value" : "Find a patient"
  }, {
    "name" : "resultFormat",
    "value" : "allure2"
  } ],
  "parameters" : [ {
    "name" : "errorMessage",
    "value" : "Enter 10 digits"
  }, {
    "name" : "nhsNumber",
    "value" : "123456789"
  } ],
  "links" : [ ],
  "hidden" : false,
  "retry" : false,
  "extra" : {
    "severity" : "normal",
    "retries" : [ ],
    "categories" : [ {
      "name" : "Product defects",
      "matchedStatuses" : [ ],
      "flaky" : false
    } ],
    "tags" : [ ]
  },
  "source" : "ec8a50870930f523.json",
  "parameterValues" : [ "Enter 10 digits", "123456789" ]
}